core/domain/question_domain.py

```diff
@@ -1863,7 +1863,7 @@ def partial_validate(self) -> None:
         if interaction.default_outcome.labelled_as_correct:
             at_least_one_correct_answer = True
 
-        if interaction.default_outcome.dest is not None:
+        if interaction.default_outcome.dest:
```

Comment:
Please don't do this -- the existing code in the backend is correct and questions should still be held to that standard of validation.

Instead, please include @JeeveshGarg's changes from #17199 in the frontend, that should help fix the error.

Reply:
Thanks. I followed @JeeveshGarg's changes and the codes can work well.